web/pdf_page_view.js

```diff
@@ -496,10 +496,10 @@ var PDFPageView = (function PDFPageViewClosure() {
     beforePrint: function PDFPageView_beforePrint() {
       var pdfPage = this.pdfPage;
 
-      var viewport = pdfPage.getViewport(1);
+      var viewport = pdfPage.getViewport(2);
       // Use the same hack we use for high dpi displays for printing to get
       // better output until bug 811002 is fixed in FF.
-      var PRINT_OUTPUT_SCALE = 2;
+      var PRINT_OUTPUT_SCALE = 1;
```

Comment:
If we're changing this to 1 we should just remove everything related to PRINT_OUTPUT_SCALE since it now does nothing.